frontend/src/pages/dashboard.tsx

```diff
@@ -64,39 +43,37 @@ const Dashboard: NextPage = (): JSX.Element => {
                 <h2 className="ml-6 font-semibold">Dashboard</h2>
                 <div className="flex items-center">
                   <div className="flex-shrink-0">
-                    <Link href={`/profile/${author?.id}`}>
+                    <Link href={`/profile/${data?.data?.user?.id}`}>
```

Comment:
why is this `data.data`?
```suggestion
                    <Link href={`/profile/${data.data.user?.id}`}>
```

Reply:
`swr` by default has a `data` value and also laravel api should have a default `data` prop in order to get the api data through it

![image](https://user-images.githubusercontent.com/108642414/187360448-cb03da1f-30e0-4bc6-bdf6-4518a4220440.png)
![image](https://user-images.githubusercontent.com/108642414/187360508-3b8cb3f8-6776-4b77-8297-8b03baf3678f.png)

Reply:
Can you destructure the inner `data` instead of accessing it from the swr `data` constant? 

`const { data } = data`

then access user id from there